backend/plugin/parser/mysql/affected_rows.go

```diff
@@ -0,0 +1,443 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/antlr4-go/antlr/v4"
+	mysql "github.com/bytebase/mysql-parser"
+	"github.com/pkg/errors"
+
+	storepb "github.com/bytebase/bytebase/proto/generated-go/store"
+)
+
+// GetStatementType return the type of statement.
+func GetStatementType(stmt *ParseResult) (result string) {
```

Comment:
Just return in the `for`s, and return "UNKNOWN" at the end.

Reply:
will do